pkg/stack/tile_stack.go

```diff
@@ -0,0 +1,66 @@
+package stack
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type Tile struct { // To be replaced by #9
+	id int
+}
+
+type Stack struct {
+	seed    int64
+	turn_no int32
+	tiles   []Tile
+	order   []int32
+}
```

Comment:
It won't be able to function as a discard pile? I mean it doesn't have any function for inserting new tile. Or am i overcomplicating?